docs/BR.md

```diff
@@ -957,11 +963,11 @@ The Random Value SHALL remain valid for use in a confirming response for no more
 
 ##### 3.2.2.5.6 ACME “http-01” method for IP Addresses
 
-Confirming the Applicant's control over the IP Address by performing the procedure documented for an “http-01” challenge in draft 04 of “ACME IP Identifier Validation Extension,” available at <https://tools.ietf.org/html/draft-ietf-acme-ip-04#section-4>.
+Confirming the Applicant's control over the IP Address by performing the procedure documented for an “http-01” challenge in RFC 8738.
```

Comment:
The only potential normative requirement change I see with this is that the RFC specifies a particular format for IPv4 and IPv6 addresses in the Host header, whereas the -04 draft does not: https://author-tools.ietf.org/iddiff?url1=draft-ietf-acme-ip-04&url2=rfc8738&difftype=--html.

I doubt this is material though, since implementations will already be using the RFC-prescribed format.

Reply:
Also agree. Same as above